### PR TITLE
Add test database setup for BigQuery.

### PR DESCRIPTION
**Pull request by tswast**
This allows you to run ./ci/datamgr.py bigquery to create a test
database for the BigQuery integrations. You no longer need to access the
ibis-gbq GCP project to run the tests.

### CODE REVIEW

**Comment by cpcloud** [comment]
Most excellent. Thank you sir. The CI failure looks like it might be transient. I'll try restarting.

File: docs/source/developer.rst [hunk 1]
```diff
@@ -101,6 +101,30 @@ Impala (with UDFs)
 
       ci/impalamgr.py load --data --data-dir ibis-testing-data
 
+BigQuery
+^^^^^^^^
+
+Before you begin, you must have a `Google Cloud Platform project
+<https://cloud.google.com/docs/overview/#projects>`__ with billing set up the
```
Comment (cpcloud) [inline docs/source/developer.rst]: Did you mean `and the BigQuery API enabled`?

Comment (tswast) [reply docs/source/developer.rst]: Yes. Fixed in https://github.com/ibis-project/ibis/pull/1523/commits/14eb0ca170bc89693410b62b1ea1bda78329a647

**Comment by cpcloud** [comment]
@tswast Coverage is a bit misleading here, likely because BigQuery tests were skipped. The tests will be run on master, so that's fine. If you want to setup your creds on your personal Circle then the coverage will more accurately reflect the actual state of the world, but totally up to you :)

**Comment by cpcloud** [comment]
thanks @tswast!

**Pull request closed without merging**